queue_services/entity-emailer/src/entity_emailer/email-templates/common/business-info.html

```diff
@@ -0,0 +1,53 @@
+<!-- table and cell padding doesn't work consistently -- use cellspacing instead -->
+<table class="business-info-table" cellspacing="10" role="presentation">
+  <tr>
+    <td class="name">Company Name:</td>
+    {% if incorporationApplication.nameRequest %}
```

Comment:
This may not work with the new change you put in yesterday as it is present for both named and numbered company. Check specifically for nrNumber inside name request to know whether it is a named company

Reply:
Good catch. I will update it!